form.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+    function func(String $text, int $size, &$valid, &$errors){
```

Comment:
Please give a meaningful name to the function. This way you will not need to read the definition of the function in order to understand the purpose of it, especially if the usage is read by somebody else later.